### DISCUSSION TITLE
Question on security onion, elastic permissions

### DISCUSSION THREAD

**Discussion by killmasta93** [Security-Onion-Solutions/securityonion#12707] [Unsupported Versions]
Hi
I was wondering if someone could shed some light, Currently installed the elastic agent, with the Elastic defender policy, but when i try to go to the url i keep getting permission denied not sure what else i missed? 

https://192.168.7.225/kibana/app/security/administration/endpoints

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13953629/73bf1067-8352-4ca6-a12e-d70e012ab8e3)

**Comment by cm-ops** [comment]
We hide that in the default space, but you might need to configure security privledges, for that `Endpoint List`. https://www.elastic.co/guide/en/security/current/endpoint-management-req.html

**Reply by killmasta93** [reply]
@cm-ops thanks for the reply, i was trying to follow this guide https://www.youtube.com/watch?v=KuzYA_0zHkY
which shows the part of the defend,
i was checking the link, but does not let me edit the kibana admin user, 
would i need to create another user? and copy the role then add the permissions? 
Thank you
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13953629/1f6351dc-060e-40e5-8dda-d875f0b83d8d)

**Reply by cm-ops** [reply]
In the video he is using the Elastic Defend on a standalone Elastic install, not Sercurity Onion. https://docs.securityonion.net/en/2.4/elastic-fleet.html#elastic-defend-endpoints-elastic-defend-integration is in the endpoints-initial policy and you will see logs in Security Onion which you can use to build detections on.

**Reply by killmasta93** [reply]
thanks for clearing that up for me
